modules/shared/components/atoms/avatar/index.tsx

```diff
@@ -0,0 +1,83 @@
+import type { FC, ReactElement } from 'react';
+import className from 'classnames';
+import React from 'react';
+import Image from 'next/image';
+import type { IAvatar } from './IAvatar';
+
+export const determineImageSize = (componentSize: string): number => {
+  const smallSize = 32;
+  const mediumSize = 40;
+  const largeSize = 56;
+  if (componentSize === 'small') {
+    return smallSize;
+  }
+  if (componentSize === 'medium') {
+    return mediumSize;
+  }
+  return largeSize;
+};
+
+const Index: FC<IAvatar.IProps> = ({ size, variant, imgSrc }): ReactElement => {
```

Comment:
we need to use a meaningful name for the component, It will make the code more readable, and easier to debug in react dev tools.

Reply:
I forgot to change the name :)